docs/howto/catalog-queries.md

```diff
@@ -12,6 +12,10 @@ Then you can query the catalog by using `curl` commands and the `jq` CLI tool to
     By default, Catalogd is installed with TLS enabled for the catalog webserver.
     The following examples will show this default behavior, but for simplicity's sake will ignore TLS verification in the curl commands using the `-k` flag.
 
+!!! note 
+    While using the `/api/v1/metas` endpoint shown in the below examples, it is important to note that the endpoint accepts parameters following the pattern `/api/v1/metas?<parameters>`. Valid parameter set is a set that corresponds to any combination of the fields of the [FBC Meta structure](https://olm.operatorframework.io/docs/reference/file-based-catalogs/#schema) (except for `Blob`).
+    eg. `schema=<schema_name>&package=<package_name>`, `schema=<schema_name>&name=<name>`, `package=<package_name>&name=<name>` are all valid paramet combination. However `schema=<schema_name>&version=<version_string>` is not a valid parameter combination, since version is not a first class FBC meta field. 
```

Comment:
```suggestion
    e.g. `schema=<schema_name>&package=<package_name>`, `schema=<schema_name>&name=<name>`, and `package=<package_name>&name=<name>` are all valid parameter combinations. However `schema=<schema_name>&version=<version_string>` is not a valid parameter combination, since version is not a first class FBC meta field. 
```